src/interface/src/app/home/about/about.component.html

```diff
@@ -1,26 +1,25 @@
-<h1>A new wildfire resilience planning tool</h1>
+<h1>Planscape is a new wildfire resilience planning tool</h1>
 
 <h3>
-  designed to bring the best available state & federal data and science
-  together. Planscape guides regional planners in prioritizing landscape
-  treatments to mitigate fire risk, maximize ecological benefits, and help
-  California's landscapes adapt to climate change.
+  Designed to bring the best available state & federal data and science
+  together, Planscape guides regional planners in identifying the optimal
+  landscapes to treat in order to mitigate fire risk, maximize ecological
+  benefits, and help California's landscapes adapt to climate change.
 </h3>
 
 <ul>
   <li *ngFor="let bullet of bulletPoints">
     <h3>{{ bullet }}</h3>
   </li>
+  <li></li>
 </ul>
 
 <div class="data-info-box">
   <p>
     The data used is based on California's Regional Resource Kits, developed by
     the California Wildfire & Forest Resilience Task Force. You can find more
     detailed information about these
-    <a href="https://wildfiretaskforce.org/regional-resource-kits-page/#kits">
-      here
-    </a>
-    .
+    <!-- eslint-disable-next-line prettier/prettier -->
```

Comment:
I can't seem to get the right format here...prettier keeps doing this:
```
  <p>
    The data used is based on California's Regional Resource Kits, developed by
    the California Wildfire & Forest Resilience Task Force. You can find more
    detailed information about these
    <a href="https://wildfiretaskforce.org/regional-resource-kits-page/#kits">
      here
    </a>
    .
  </p>
```

Which adds an unwanted space between the end of the link and the `.`